backends/ebpf/ebpfstf.py

```diff
@@ -82,7 +82,7 @@ def create_table_file(actions, tmpdir, file_name):
     the control plane. """
     with open(tmpdir + "/" + file_name, "w+") as control_file:
         control_file.write("#include \"test.h\"\n\n")
-        control_file.write("static inline void generated_init() {\n\t")
+        control_file.write("static inline void run_generated_cmds() {\n\t")
```

Comment:
I think we should name this function "initialize_tables".
There are no commands really at this level.